wallet/src/Pos/Wallet/Web/Methods/Logic.hs

```diff
@@ -164,14 +126,21 @@ getAccounts = getAccountsIncludeUnready False
 
 getWalletIncludeUnready :: MonadWalletWebMode m => Bool -> CId Wal -> m CWallet
 getWalletIncludeUnready includeUnready cAddr = do
```

Comment:
`cAddr = cWalId` ? Not sure what the convention is, but `WalletId` is more clear than `Addr`.

Reply:
Ok, remaned